pkg/internal_registry/registry.go

```diff
@@ -0,0 +1,57 @@
+package registry
+
+import (
+	brokerModel "github.com/eclipse/che-plugin-broker/model"
+)
+
+func InternalRegistry() map[string]brokerModel.PluginMeta {
+
+	return map[string]brokerModel.PluginMeta{
+		"eclipse/cloud-shell/nightly": {
+			APIVersion:  "v2",
+			Publisher:   "eclipse",
+			Description: "Cloud Shell provides an ability to use terminal widget like an editor.",
+			DisplayName: "Cloud Shell Editor",
+			ID:          "",
```

Comment:
I think ID needs to be set as well -- I'm seeing
```
containers:
        che-machine-exec:
          attributes:
            plugin: ""
            source: tool
          ports:
          - 4444
```
in the component description, which looks wrong.

Reply:
c.f.:
```yaml
      containers:
        che-machine-exec:
          attributes:
            plugin: eclipse/cloud-shell/nightly
            source: tool
          ports:
          - 4444

```
in master (though this has no bearing on cloud-shell)

Reply:
Found where `getPluginMetadata` sets ID explicitly: https://github.com/eclipse/che-plugin-broker/blob/master/utils/metadata_resolver.go#L65-L73